src/features/customerView/CustomerViewContainer.tsx

```diff
@@ -244,8 +244,8 @@ const CustomerViewContainer = () => {
 
       <Modal isOpen={!!openResendInvoice} toggleModal={() => setOpenResendInvoice(undefined)}>
         <SendInvoiceFormContainer
-          orderId={openResendInvoice?.orderId ?? ''}
-          email={customerProfile.primaryEmail ?? null}
+          orders={[{ orderId: openResendInvoice?.orderId ?? '', email: openResendInvoice?.customer.email ?? null }]}
+          // email={customerProfile.primaryEmail ?? null}
```

Comment:
What's up with this?

Reply:
I removed these commented props, I will have to have a discussion with Joonas or Kalle regarding the situation of missing emails since now this form supports multiple offers